### ISSUE TITLE
power on/off support

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#27]
```
Show player power status & add power on/off controls.

```

Original issue reported on code.google.com by `bradfitz` on 2009-08-18 03:46:04

**Comment by kaaholst** [comment]
```
I have implemented power on/off functionality for the players which supports it.
The player can be turned on or off via an apropriate menu option, depending on wheater
the player is currently turned on.

I have enclosed a patch with the new functionality.
```

Original issue reported on code.google.com by `kaaholst` on 2010-11-02 22:13:08

<hr>
- _Attachment: [powerctrl.patch](https://storage.googleapis.com/google-code-attachments/android-squeezer/issue-1/comment-1/powerctrl.patch)_

**Comment by kaaholst** [comment]
```
Excellent, thanks!
```

Original issue reported on code.google.com by `bradfitz` on 2010-11-04 23:24:39

**Comment by kaaholst** [comment]
```
OOPS, I forgot to mention that the patch is based on the second latest version of the
code, ie. the one which corresponds to squeezer version 0.4 from androlib. 
I'll be happy to use the latest git version, but then I need a few hints on how to
complete that version, because it is work in progress.

```

Original issue reported on code.google.com by `kaaholst` on 2010-11-05 07:25:37

**Comment by kaaholst** [comment]
```
Committed:

https://github.com/bradfitz/android-squeezer/commit/5c73633d24201565832ab9e2ca6980523275092b

And released 0.5 to the Android Market.

Thanks again!

```

Original issue reported on code.google.com by `bradfitz` on 2010-11-05 19:39:02
